cloud-sql/postgres/sqlalchemy/connect_connector.py

```diff
@@ -0,0 +1,103 @@
+"""
+Copyright 2022 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  https://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+# [START cloud_sql_postgres_sqlalchemy_connect_connector]
+import os
+
```

Comment:
What is the significance of these newlines? Standard lib, connector, external?

Reply:
The lint job has it set that it wants new lines between the different type of imports (std lib vs Application vs Third Party)